docs/custom-flows/passkeys.mdx

```diff
@@ -0,0 +1,233 @@
+---
+title: Passkeys
+description: Learn how to allow your users authenticate with passkeys with Clerk.
+---
+
+# Passkeys
+
+<Callout type="Info">
+  Passkey user sign-in is available on Pro plans. [Upgrade](/pricing) to enable this feature.
+</Callout>
+
+A passkey is a type of sign-in credential that requires one user action, but uses two authentication factors:
+  1. A pin number or biometric data
+  1. A physical device
+
+Because of this, passkeys are generally more secure than passwords, and you should allow users to sign into your Clerk app with them. You can do so with the prebuilt [`<SignIn>`](/docs/components/authentication/sign-in) component, or with Clerk's JS SDK.
+
+<Callout type="Info">
+  Users can't create passkeys with the `<SignUp>` component.
+</Callout>
+
+## Enable passkeys in your dashboard
+
+To use passkeys, you must first enable the feature in your Clerk Dashboard:
+
+1. Visit the Dashboard and select your project.
+1. Expand **User & Authentication**, then select [**Email, Phone, Username**](https://dashboard.clerk.com/last-active?path=user-authentication/email-phone-username).
+1. Navigate to **Authentication strategies**, and toggle **Passkeys**.
+
+## Manage user passkeys
+
+If you're using the prebuilt [`<UserProfile>`](/docs/components/user/user-profile) component, your users can create and manage their passkeys in the **Security** section of their [account portal](/docs/account-portal/overview#account-portal-overview).
+
+The following instructions will teach you how to enable this functionality with a custom UI.
```

Comment:
Move to ### Passkeys on this page /docs/authentication/configuration/sign-up-sign-in-options

```suggestion
```